src/actions/addons/addons.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Add-ons actions.
+ * @module actions/addons/addons
+ */
+
+import {
+  INSTALL_ADDON,
+  LIST_ADDONS,
+  UNINSTALL_ADDON,
+} from '../../constants/ActionTypes';
+
+/**
+ * List addons function
+ * @function listAddons
+ * @returns {Object} List addons action
+ */
+export function listAddons() {
+  return {
+    type: LIST_ADDONS,
+    request: {
+      op: 'get',
+      path: `/@addons`,
+    },
+  };
+}
+
+/**
+ * Install addon function.
+ * @function installAddon
+ * @param {string} id Addon id
+ * @returns {Object} install addon action.
+ */
+export function installAddon(id) {
+  return {
+    type: INSTALL_ADDON,
+    promise: api => api.post(`/@addons/${id}/install`, { data: { id } }),
```

Comment:
this is the "old" method of doing a backend call which we changed recently. You need to do the request with a request object like you did in `listAddons` so:
```
request: {
  op: 'post',
  path: `/@addons/${id}/install`,
  data: { id },
}
```